portfolio/src/main/java/com/google/sps/servlets/StringServlet.java

```diff
@@ -8,30 +8,32 @@
 
 
 
-@WebServlet("/strservlet")
+@WebServlet("/strServlet")
 public class StringServlet extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         response.setContentType("text/String;");
         Gson gson = new Gson();
         
-        //create a new array list and input quoataions.
+        // create a new array list and input quoataions.
         ArrayList<String> quotations = new ArrayList<String>();
 
-        quotations.add("<p>The past is not dead. In fact, it's not even past.  -William Faulkner</p>");
-        quotations.add("<p>Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X</p>");
-        quotations.add("<p>Don't count the days, make the days count. -Muhammad Ali</p>");
+        quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
+        quotations.add("Education is the passport to the future," 
+                          +" for tomorrow belongs to those who prepare for it today. -Malcolm X");
```

Comment:
nit: Need a space between operators too. Also the line continuation seems off (should be indented by 4 spaces from the previous line).

Reply:
Ok, gotcha. Just fixed those and will merge.